noaa.go

```diff
@@ -98,16 +98,18 @@ type ForecastResponsePeriod struct {
 
 // ForecastResponsePeriodHourly provides the JSON value for a period within an hourly forecast.
 type ForecastResponsePeriodHourly struct {
-	ID               int32  `json:"number"`
-	Name             string `json:"name"`
-	StartTime        string `json:"startTime"`
-	EndTime          string `json:"endTime"`
-	IsDaytime        bool   `json:"isDaytime"`
-	TemperatureTrend string `json:"temperatureTrend"`
-	WindDirection    string `json:"windDirection"`
-	Summary          string `json:"shortForecast"`
-	Icon             string `json:"icon,omitempty"`
-	Details          string `json:"detailedForecast"`
+	ID              int32   `json:"number"`
+	Name            string  `json:"name"`
+	StartTime       string  `json:"startTime"`
+	EndTime         string  `json:"endTime"`
+	IsDaytime       bool    `json:"isDaytime"`
+	Temperature     float64 `json:"temperature"`
```

Comment:
It looks like they are in the process of redesigning the temperature/temperature units in the API. Since the hourly forecast endpoint returns all three values (temp, temp unit, and trend) maybe we should keep them all for now until and unless they break the public-facing API. See also: https://api.weather.gov/openapi.yaml